src/i18n/en.po

```diff
@@ -151,15 +151,15 @@ msgstr ""
 msgid "Add a label to quickly find the transaction in your history."
 msgstr ""
 
-#: src/components/modals/SendModal.vue:174
+#: src/components/modals/SendModal.vue:176
```

Comment:
We usually try to exclude the language files from PRs, as they only create merge conflicts. They are rebuild when the app is deployed.

Reply:
:see_no_evil: